modules/standard/Random.chpl

```diff
@@ -293,8 +337,21 @@ module Random {
         compilerError("NPBRandomStream(eltType=", eltType:string, 
                       ") can only be used to fill arrays of ", eltType:string);
       }
-        
-      pragma "no doc"
+
+      /*
+         Returns an iterable expression for generating
+         D.numIndices random numbers. The RNG state will
+         be immediately advanced by D.numIndices before
+         this function yields any values.
+
+         The iterable expression can be usefully used
+         in a parallel context, whether standalone or
+         zippered.
+
+         :arg D: a domain
+         :arg resultType: the type of number to yield
+         :return: an iterable expression yielding random resultType values
+       */
```

Comment:
You're correct that I was unaware that the random module primer exposed this method (weeps silently).